packages/aws-cdk-lib/aws-ecs/lib/amis.ts

```diff
@@ -97,6 +97,13 @@ export interface EcsOptimizedAmiProps {
    * @default false
    */
   readonly cachedInContext?: boolean;
+
+  /**
+   * What kernel version of Amazon Linux 2 to use
+   *
+   * @default none, uses the recommended kernel version
+   */
+  readonly kernel?: ec2.AmazonLinux2Kernel;
```

Comment:
```suggestion
  /**
   * Kernel version used.
   * Only supported for Amazon Linux 2.
   *
   * @default none, uses the recommended kernel version
   */
  readonly kernel?: string;
```
What if we need to support the property for other Linux images in the future? 🤔
```

Reply:
That's better. Thanks a lot 🙏